kolibri/core/utils/sqlalchemy.py

```diff
@@ -0,0 +1,39 @@
+from django.db import connections
+from sqlalchemy.pool import NullPool
```

Comment:
Would [`from __future__ import absolute_import`](https://github.com/learningequality/kolibri/blob/develop/kolibri/core/tasks/apps.py#L1) potentially cause this?
I notice the same error here: https://stackoverflow.com/a/33744089

Reply:
Specifically because the file is the same name of the module

Reply:
That's the ticket!